### ISSUE TITLE
Alternative Printing: Spaces in printer name cause failure

### ISSUE THREAD

**Issue by macarenomarco** [qzind/tray#20]
I'm currently running
- OS X 10.10.5
- CUPS 2.0.0
- QZ Tray 2.0.0
- `java -version` output:
```
    java version "1.8.0_60"
    Java(TM) SE Runtime Environment (build 1.8.0_60-b27)
    Java HotSpot(TM) 64-Bit Server VM (build 25.60-b23, mixed mode)
```
- Zebra ZTC_GX420t

When attempting to print (using `altPrint: true`) I ran into:  
`Alternate printing returned a non-zero value(1).`

After some directly logging the command being used if found out it was using the following:
`/usr/bin/lp -d "Zebra Technologies ZTC GX420t" -o raw "/var/folders/vw/t8cm364n2q51d_hsrwqpcdp40000gn/T/qz_raw_4797437088864693128.tmp";`

And after manually attempting to run it, it would output the following:
`lp: The printer or class does not exist.`   
Which was causing the non-zero return value.
It all came down to the fact that while the javax `PrintServiceLookup` was returning the name `Zebra Technologies ZTC GX420t`  
The output of `lpstat` was the following: 
```
printer KONICA_MINOLTA_bizhub_C35_DB_7E_F9_ is idle.  enabled since Wed Sep 30 15:33:22 2015
printer Samsung_M2020_Series__SEC30CDA7AFF793_ is idle.  enabled since Wed Mar 23 20:10:45 2016
printer Zebra_Technologies_ZTC_GX420t is idle.  enabled since Wed Mar 30 20:52:49 2016
	Sending data to printer.
system default destination: Zebra_Technologies_ZTC_GX420t
```
It seems to be that `PrintServiceLookup` interprets the spaces correctly while `lpstat` changes them to underscores.

**Comment by tresf** [comment]
Good find and thanks for the research.

Here's a more thorough explanation, scroll to the **Exceptions** section. https://github.com/qzind/tray/pull/17

We'll leverage the new [`getCupsPrinters()`](https://github.com/qzind/tray/pull/17/files#diff-a3769499f474da162b4d0ad0089105e4R169) shell utility, once merged and patch the alternate printing method as well.

**Comment by macarenomarco** [comment]
All right, thanks c:

**Comment by tresf** [comment]
Fixed via 3680d9f.  This will be available in the next release (either RC4 or 2.0.0 official)
